adyenv6core/src/com/adyen/v6/converters/PosPaymentResponseConverter.java

```diff
@@ -0,0 +1,54 @@
+/*
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Hybris Extension
+ *
+ * Copyright (c) 2019 Adyen B.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ */
+package com.adyen.v6.converters;
+
+import com.adyen.model.checkout.PaymentsResponse;
+import com.adyen.model.nexo.SaleToPOIResponse;
+import org.springframework.core.convert.converter.Converter;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class PosPaymentResponseConverter implements Converter<SaleToPOIResponse, PaymentsResponse> {
```

Comment:
Do we need this class ?

Reply:
This was added to convert the terminal API response to a PaymentsResponse, so in that way we can reuse the method `DefaultAdyenCheckoutFacade.createAuthorizedOrder(paymentsResponse)` and create the order using the same flow as in the checkout authorisation.